test/createFunction/FunctionTesterBase.ts

```diff
@@ -27,7 +27,7 @@ suiteSetup(async function (this: IHookCallbackContext): Promise<void> {
     // Ensure template data is initialized before any 'Create Function' test is run
     backupTemplateData = await getTemplateDataFromBackup(undefined, path.join(__dirname, '..', '..', '..'));
     funcPortalTemplateData = await tryGetTemplateDataFromFuncPortal(undefined);
-    funcStagingPortalTemplateData = await tryGetTemplateDataFromFuncPortal(undefined, undefined, 'functions-staging.azure.com');
+    funcStagingPortalTemplateData = await tryGetTemplateDataFromFuncPortal(undefined, undefined);
```

Comment:
At least file an issue on this because right now it's just going to run the tests twice on the same templateData which is pointless. Ideally 'funcStagingPortalTemplateData' tests against the pre-release of the templates